environments/extractDiffs.toml

```diff
@@ -87,6 +87,10 @@
         deployed = 10790528
+    [contract.MCD_FLIP_RENBTC_A_1_2_1]
```

Comment:
I just realized that maybe we don't want to include these in here until the backfillEvents process has been completed, because otherwise we'll probably end up with some diffs that we can't transform yet? But on the other hand, the sooner we get this in, the less backfill we'll have to do for storage diff. 🤔